pkg/datasources/databases_acceptance_test.go

```diff
@@ -1,105 +1,68 @@
 package datasources_test
 
 import (
-	"fmt"
 	"strconv"
 	"testing"
 
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider/resources"
+	"github.com/hashicorp/terraform-plugin-testing/config"
+
 	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
 
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
-	"github.com/hashicorp/terraform-plugin-testing/terraform"
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
-func TestAcc_Databases(t *testing.T) {
+func TestAcc_Databases_Complete(t *testing.T) {
```

Comment:
should we add tests for:
- some filtering
- these checks that were added to examples
?